src/Templates/MLDotNet/Microsoft.VisualStudio.Templates.MLDotNet/Content/ConsoleApp/Program.cs

```diff
@@ -0,0 +1,87 @@
+/* This template shows the building blocks for training a machine learning model with ML.NET (https://aka.ms/mlnet).
+ * This model predicts whether a sentence has a positive or negative sentiment. It is based on a sample that can be 
+ * found at https://aka.ms/mlnetsentimentanalysis, which provides a more detailed introduction to ML.NET and the scenario. */
+
+using System;
+using System.IO;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML;
+using Microsoft.ML.Runtime.Api;
+using Microsoft.ML.Transforms.Text;
+
+namespace $safeprojectname$
+{
+    class Program
+    {
+        static void Main()
+        {
+            // 1. Build an ML.NET pipeline for training a sentiment analysis model.
+            Console.WriteLine("Training a model for Sentiment Analysis using ML.NET");
+            var mlContext = new MLContext();
+            // 1a. Load the training data using a TextLoader.
+            var reader = 
+                new TextLoader(
+                    mlContext,
+                    new TextLoader.Arguments()
+                    {
+                        Separator = "tab",
+                        HasHeader = true,
+                        Column = new[]
+                        {
+                            new TextLoader.Column("Label", DataKind.Bool, 0),
+                            new TextLoader.Column("Text", DataKind.Text, 1)
+                        }
+                    });
+
+            IDataView trainingDataView = reader.Read(new MultiFileSource(@"Data\wikipedia-detox-250-line-data.tsv"));
+            // 2. Create a pipeline to prepare your data, pick your features and apply a machine learning algorithm.
+            // 2a. Featurize the text into a numeric vector that can be used by the machine learning algorithm.
+            var pipeline = mlContext.Transforms.Text.FeaturizeText("Text", "Features")
+                    .Append(mlContext.BinaryClassification.Trainers.StochasticDualCoordinateAscent( "Label", "Features"));
+
+            // 3. Get a model by training the pipeline that was built.
+            var model = pipeline.Fit(trainingDataView);
+
+            // 4. Evaluate the model to see how well it performs on different data (output the percent of examples classified correctly).
+            Console.WriteLine("Training of model is complete \nTesting the model with test data");
+            IDataView testDataView = reader.Read(new MultiFileSource(@"Data\wikipedia-detox-250-line-test.tsv"));
+            var predictions = model.Transform(testDataView);
+            var binClassificationCtx = new BinaryClassificationContext(mlContext);
```

Comment:
You shouldn't be creating a `BinaryClassificationContext`. Instead you should get it from the `mlContext`.